src/NzbDrone.Core/Parser/ParsingService.cs

```diff
@@ -80,15 +70,6 @@ public ParsedMovieInfo EnhanceMovieInfo(ParsedMovieInfo minimalInfo, List<object
                 minimalInfo = AugmentMovieInfo(minimalInfo, helpers);
             }
 
-            // minimalInfo.Quality.Quality = QualityFinder.FindBySourceAndResolution(minimalInfo.Quality.Quality.Source, minimalInfo.Quality.Quality.Resolution,
```

Comment:
Do we even need this EnhanceMovieInfo method anymore? What is the parser AugmentMovieInfo doing different than the movieFile Augmenter?

Reply:
Yes, looks like it does - this is how the indexer flags and a few other bits flow through so not sure about changing this just yet